src/components/Transaction/TransactionCard/index.tsx

```diff
@@ -108,17 +80,18 @@ const TransactionCard = ({
 }) => {
   return (
     <CardPanel>
-      <CardLabelItem
-        value={`${startEndEllipsis(transaction.transaction_hash, 14)}`}
-        to={`/transaction/${transaction.transaction_hash}`}
```

Comment:
keep the function of jumping to "Transaction Page" while refactoring the code

Reply:
Ok, I have fixed it.  https://github.com/nervosnetwork/ckb-explorer-frontend/pull/74/commits/6eb1f1807679db3e17a0490399ddae4b507ee1ed